static/js/fixBrokenSceneMarks.js

```diff
@@ -12,6 +12,13 @@ var FIX_BROKEN_SCENE_MARK_PARAM = 'fixBrokenSceneMarks';
 var UNDERLINE_ATTRIB = 'underline';
 var SCRIPT_STRUCTURE = 'script structure'; // message sent to Google Analytics
 
+// fixed lines are marked with red font and are underlined
+var UNDERLINE = [UNDERLINE_ATTRIB, true];
+var RED_COLOR = [epFontColorUtils.FONT_COLOR_ATTRIB_KEY, epFontColorUtils.RED_COLOR];
+var FIXED_LINE_ATTRIBS = [RED_COLOR, UNDERLINE];
+var RED_TEXT_SELECTOR = '.' + epFontColorUtils.FONT_COLOR_ATTRIB_KEY + '-' + epFontColorUtils.RED_COLOR;
+exports.FIXED_TEXT_SELECTOR = '.u' + RED_TEXT_SELECTOR;
```

Comment:
coloca um comentário como fica o `FIXED_TEXT_SELECTOR`, por favor
```suggestion
exports.FIXED_TEXT_SELECTOR = '.u' + RED_TEXT_SELECTOR; // .u.font-color-A1
```